tests/default_environment/libnetwork_test.go

```diff
@@ -63,7 +68,7 @@ var _ = Describe("Libnetwork Tests", func() {
 			It("rejects --opt being used", func() {
 				session := DockerSession("docker network create $RANDOM --opt REJECT -d calico --ipam-driver calico-ipam")
 				Eventually(session).Should(Exit(1))
-				Eventually(session.Err).Should(Say("Error response from daemon: NetworkDriver.CreateNetwork: Arbitrary options are not supported"))
+				Eventually(session.Err).Should(Say("Error response from daemon: NetworkDriver.CreateNetwork: Calico driver does not recognize option com.docker.network.generic (map[REJECT:])"))
```

Comment:
The user visible output isn't great here. Are you sure that %v is the best way to represent the unknown "values"